apps/studio/state/storage-explorer.tsx

```diff
@@ -1281,6 +1274,10 @@ function createStorageExplorerState({
                       // Resource already exists
                       toast.error(`Failed to upload ${file.name}: File name already exists.`)
                       break
+                    case 400:
```

Comment:
realised this case wasn't being handled - can try uploading a file with paranthesis in its name e.g `Demo (Version 1)` or something